src/elements/table/table-wrapper/table-wrapper.scss

```diff
@@ -0,0 +1,33 @@
+@use '../../core/styles/index' as sbb;
+
+// Box-sizing rules contained in typography are not traversing Shadow DOM boundaries. We need to include box-sizing mixin in every component.
+@include sbb.box-sizing;
+
+:host {
+  display: block;
+}
+
+:host([data-has-horizontal-scrollbar]) {
+  ::slotted(.sbb-table) {
+    // Remove the border-radius from the bottom of the table if the scroll is visible
+    --sbb-table-border-radius: var(--sbb-border-radius-4x) var(--sbb-border-radius-4x) 0 0 !important;
```

Comment:
could we not use a solution without !important by only overriding the var? maybe as a fallback var? If not, please provide a comment on why !important is needed.

Reply:
I'm not happy with it but I believe it is the less evil solution. I'll add a comment on why it's needed